docs/source/impl-guide/normalization.rst

```diff
@@ -3,50 +3,105 @@
 Normalization
 !!!!!!!!!!!!!
 
-Certain insertion or deletion alleles may be represented ambiguously
-when using conventional sequence normalization, resulting in
-significant challenges when comparing such alleles.
+In VRS, "normalization" refers to the process of converting a given
+variation representation into a canonincal form.  Left- and
+right-shuffling are examples of normalization for sequence variants.
+VRS extends this concept to apply to all classes of VRS Variation.
+VRS normalization minimizes a class of ambiguity that impedes
+comparison of variation across systems.
 
-VRS describes a "fully-justified" normalization algorithm
-inspired by NCBI's Variant Overprecision Correction Algorithm [1]_.
+Implementations MUST provide a normalize function that accepts any
+Variation object and returns a normalized Variation object.  Guidelines for
+these functions are below.
+
+
+General Normalization Rules
+@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+* VRS normalization functions are homomorphic: That is, the input and
+  output objects are always of the same `type`.
+* Object types that do not have VRS normalization rules are returned
+  as-is.  That is, all types of VRS objects MUST be supported, even if
+  such objects are unchanged.
+* VRS normalization functions are idempotent: Normalizing a normalized
+  Variation object returns an object with the same data.
+
+
+Allele Normalization
+@@@@@@@@@@@@@@@@@@@@
+
+Certain insertion or deletion alleles may have ambiguous
+representations when using conventional sequence normalization,
+resulting in significant challenges when comparing such alleles.
+
+VRS uses a "fully-justified" normalization algorithm inspired by
```

Comment:
Does the VRS algorithm match NCBI's VOCA? 

The design_decisions.rst "Allele Are Full Justified" section has "VRS require that Alleles be fully justified from the description of the NCBI Variant Overprecision Correction".   

Is there info on normalization decision  (normalizing all VRS objects vs just VRS Variation objects) that should be added to the design decision page.?

Reply:
Yes. VOCA source code wasn't (isn't?) available, so I implemented from scratch. Brad Holmes reviewed the source and the results and agreed that they were the same thing.

The intro to normalization now explains that normalization applies to all VRS Variation objects.